library/std/tests/sync/lib.rs

```diff
@@ -0,0 +1,32 @@
+#![feature(lazy_get)]
+#![feature(mapped_lock_guards)]
+#![feature(mpmc_channel)]
+#![feature(once_cell_try)]
+#![feature(once_wait)]
```

Comment:
error: the feature `once_wait` has been stable since 1.86.0-dev and no longer requires an attribute to enable
 --> library/std/tests/sync/lib.rs:5:12
  |
5 | #![feature(once_wait)]
  |            ^^^^^^^^^
  |
  = note: `-D stable-features` implied by `-D warnings`
  = help: to override `-D warnings` add `#[allow(stable_features)]`